### PR TITLE
Fix Server Side rendering support using suggested PR from fixed-data-table

### PR DESCRIPTION
**Pull request by wcjordan**
## Description
have some state that tells us we are in the first pass and always render positions with left and top (since we know all browsers support this but not translate pos), then on any other render from there we can use the fancy translate3d stuff

## Motivation and Context
Server side rendering currently produces different markup than client rendering
https://github.com/schrodinger/fixed-data-table-2/issues/11

## How Has This Been Tested?
I haven't tried this out.  I'll verify everything still works with client side rendering.
We should put together a server side example / sandbox for testing

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: src/FixedDataTableTranslateDOMPosition.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
```
Comment (KamranAsif) [inline src/FixedDataTableTranslateDOMPosition.js]: Needs to be updated to our new copyright
```
/**
 * Copyright Schrodinger, LLC
 */
```

File: src/FixedDataTableBufferedRows.react.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ var FixedDataTableRow = require('FixedDataTableRow.react');
 var cx = require('cx');
 var emptyFunction = require('emptyFunction');
 var joinClasses = require('joinClasses');
-var translateDOMPositionXY = require('translateDOMPositionXY');
+var translateDOMPositionXY = require('FixedDataTableTranslateDOMPosition');
```
Comment (KamranAsif) [inline src/FixedDataTableBufferedRows.react.js]: nit: consider renaming to match new name

**Comment by KamranAsif** [comment]
LGTM after we verify this works with example provided here: https://github.com/schrodinger/fixed-data-table-2/issues/11

File: src/FixedDataTableBufferedRows.react.js (hunk 2 above)
Comment (wcjordan) [inline src/FixedDataTableBufferedRows.react.js]: done

File: src/FixedDataTableTranslateDOMPosition.js (hunk 1 above)
Comment (wcjordan) [inline src/FixedDataTableTranslateDOMPosition.js]: done

**Comment by wcjordan** [comment]
Verified it fixes the example.  Also verified it still works as expected in LiveDesign.
Have you been running through any other tests / samples to avoid regressions?

**Comment by KamranAsif** [comment]
I spin up the dev server ```npm run sit-dev-server``` and play around with some of the examples to make sure everything still works

**Comment by wcjordan** [comment]
Done, everything looks good

**Pull request merged**